pkg/scripts/issues/file/main.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+
+	i "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/scripts/issues"
+)
+
+func main() {
+	issues := loadIssues()
+	processedIssues := processIssues(issues)
+	saveCsv(processedIssues)
+}
+
+func loadIssues() []i.Issue {
+	bytes, err := os.ReadFile("../gh/issues.json")
```

Comment:
is this meant to load issues coming from multiple sources? if so, it seems to be strongly dependent on the github issue interface

Reply:
Nope, only from GH